tezos-sig-gating-contracts/scripts/tests/gatednftminterSimpleMultisig.spec.ts

```diff
@@ -0,0 +1,302 @@
+import { expect } from "chai";
+import { deployNFTMinterSimple } from "../fixtures/fixtureGatedNftMinterSimple";
+import { deploySignerManagerMultisig } from "../fixtures/fixtureNexeraIDSignerManagerMultisig";
+import { InMemorySigner } from "@taquito/signer";
+import {
+  MichelsonMap,
+  TezosToolkit,
+  TezosOperationError,
+} from "@taquito/taquito";
+import {
+  convert_timestamp,
+  convert_key,
+  convert_nat,
+  convert_string,
+  convert_address,
+  convert_chain_id,
+  convert_mint,
+} from "../utils/convert";
+import {
+  EdSignature,
+  TezosTxAuthData,
+  TezosTxCalldata,
+} from "../utils/schemas";
+import { buildTxCallDataNoFunctionName } from "../utils/buildTxCallData";
+import { computePayloadHash } from "../utils/computePayloadHash";
+
+const RPC_ENDPOINT = "http://localhost:20000/";
+
+const Tezos = new TezosToolkit(RPC_ENDPOINT);
+import { RpcClient } from "@taquito/rpc";
+const client = new RpcClient(RPC_ENDPOINT); //, 'NetXnofnLBXBoxo');
+
+let alice_pk = "edsk3QoqBuvdamxouPhin7swCvkQNgq4jP5KZPbwWNnwdZpSpJiEbq";
+let alice = "tz1VSUr8wwNhLAzempoch5d6hLRiTh8Cjcjb";
+let bob_pk = "edsk3RFfvaFaxbHx8BMtEW1rKQcPtDML3LXjNqMNLCzC3wLC1bWbAt";
+let bob = "tz1aSkwEot3L2kmUvcoxzjMomb9mvBNuzFK6";
+
+const nexeraSigner = new InMemorySigner(bob_pk); // signer private key
+
+describe(`GatedNftMinterSimple with SignerManagerMultisig`, function () {
+  let exampleGatedNFTMinter: string | undefined;
+  let nexeraSignerManager: string | undefined;
+  let deployerAddress: string;
+  let currentBlock: number;
+  let currentChainId: string;
+  let nexeraSignerPublicKey: string;
+  let nexeraSignerAddress: string;
+
+  before(async () => {
+    // SET SIGNER
+    deployerAddress = alice;
+    Tezos.setProvider({
+      signer: await InMemorySigner.fromSecretKey(alice_pk),
+    });
+    // Retrieve Signer public key
+    nexeraSignerPublicKey = await nexeraSigner.publicKey();
```

Comment:
so if I understand correctly bob is the signer?
if bob is also an owner of the multisig I think it would be more comprehensible if a third address was the signer address

Reply:
I added a third user (frank) who is owner in multisig and validates ; now bob is only a signer